desk/app/groups.hoon

```diff
@@ -3129,27 +3141,14 @@
       =?  cor  (~(has by foreigns) flag)
         fi-abet:(fi-watched:(fi-abed:fi-core flag) p.sign)
       ?^  p.sign
-        =.  cor  (fail:log %watch-ack 'group watch failed' u.p.sign)
-        ?.  (~(has by foreigns) flag)
-          ::TODO  this should not be possible, but if it happens
-          ::      we don't have an invitation, and thus no way to rejoin.
-          ::      the user will still see the group, but it is going
-          ::      to be stale. it would be best to somehow surface
-          ::      it at the client.
-          ::
-          =.  cor  (tell:log %crit 'misguided group watch-nack' ~)
-          go-core
-        ::  join in progress, set error and leave the group
-        ::  to allow re-joining.
-        ::
-        ::  however, do not leave the group if it was already initialized
-        ::  to avoid data loss.
+        =.  cor  (fail:log %crit 'group watch failed' u.p.sign)
```

Comment:
Isn't the `%crit`/`%watch-ack` here just a description term? Feel like that should continue to be `%watch-ack` or similar, `%crit` is one of our levels but shouldn't be used as a description for a particular trace.